localstack/aws/api/s3/__init__.py

```diff
@@ -2008,6 +2007,9 @@ class GetObjectAclRequest(ServiceRequest):
     ExpectedBucketOwner: Optional[AccountId]
 
 
+Size = int
```

Comment:
q: any idea why this parameter Type is out of the long list of parameters on top and now here?

Reply:
Ah, good catch! thanks a lot 👍